src/locate/synctex/worker.ts

```diff
@@ -91,32 +92,37 @@ function toRect(blocks: any): Rectangle {
     }
 }
 
-function parseSyncTexForPdf(pdfFile: string): PdfSyncObject | undefined {
-    const filename = path.basename(pdfFile, path.extname(pdfFile))
-    const dir = path.dirname(pdfFile)
-    const synctexFile = path.resolve(dir, filename + '.synctex')
-    const synctexFileGz = synctexFile + '.gz'
+async function parseSyncTexForPdf(pdfFile: vscode.Uri): Promise<PdfSyncObject | undefined> {
+    const filename = path.basename(pdfFile.fsPath, path.extname(pdfFile.fsPath))
+    const dir = path.dirname(pdfFile.fsPath)
+    const synctexFile = vscode.Uri.parse(pdfFile.scheme + "://" + path.resolve(dir, filename + '.synctex'))
+    const synctexFileGz = vscode.Uri.parse(pdfFile.scheme + "://" + synctexFile.fsPath + '.gz')
 
-    if (fs.existsSync(synctexFile)) {
+    try {
```

Comment:
Do not wrap with try-catch. Use `lw.file.exists` to check file existence.

Reply:
I think Uri includes remote File, so there is not exists method so it have to try open file and catch when the file is not exists.